### PR TITLE
Ped cancer

### PR DESCRIPTION
**Pull request by CarolinePotteiger**

### CODE REVIEW

File: README.md
Comment (CarolinePotteiger) [inline README.md:1]: I think this new link is more up to date?

File: input/pagecontent/index.md
Comment (CarolinePotteiger) [inline input/pagecontent/index.md:1]: Changed authors of mCODE to be in alphabetical order

**Comment by saulakravitz** [comment]
You can get rid of the errors in your CapStmts by deleting the text.div and text.generated from all of them.
These are no longer needed now that we have the liquid template version of CapSTmt rendering in place.

**Comment by saulakravitz** [comment]
Recommend you use a ruleset for your Bundle example.  You have some type/instance mismatches that I find easier to sight in a more concise FSH.

Observation <==> us-core-procedure-biopsy-jenny-m

My go to is:
RuleSet: addentry(type, id)
* entry[+].resource = {id}
* entry[=].fullUrl = "http://www.example.org/fhir/{type}/{id}"

File: input/images/mCodeDiagram.svg
Comment (saulakravitz) [inline input/images/mCodeDiagram.svg:1]: Links from some of the elements don't work.  I think because they don't point to the build version on your branch.   Tumor Morphology points to http://hl7.org/fhir/us/mcode/StructureDefinition-mcode-tumor-morphology.html

Is the link between Tumor Morphology (and Histologic Grade) <--> Histologic Behavor and Type really bidrectional?
Looks like TumorMorphologyReport.result --> HistologicBehaviorAndType, but not v.v. 
Same question for Cancer Risk Assessment and Cancer Condition.

Shouldn't BodySurfaceArea be included among external profiles?

File: input/pagecontent/change_log.md
Comment (saulakravitz) [inline input/pagecontent/change_log.md:8]: Broken link

File: input/pagecontent/change_log.md
Comment (saulakravitz) [inline input/pagecontent/change_log.md:18]: I wouldn't say this is a deficit of STU3, but rather that Risk Assessments were recognized as essential for representing the case data for a Cancer Patient.

File: input/pagecontent/change_log.md
Comment (saulakravitz) [inline input/pagecontent/change_log.md:30]: The title of [TumorMorphology] is 'Tumor Morphology Report'. 
I recommend changing the computable name of the profile to align with the title, since the misalignments require cognitive energy to resolve.  It is a report on Tumor Morphology.

File: input/pagecontent/change_log.md
Comment (saulakravitz) [inline input/pagecontent/change_log.md:38]: Again, I wouldn't state this as an mCODE STU3 deficit, but that with the focus of STU4 on pediatric cancer, Lansky was recognized as important to support.

File: input/pagecontent/change_log.md
Comment (saulakravitz) [inline input/pagecontent/change_log.md:60]: I think all of the material here can be deleted.  At the top you say "go look at the change log in the relevant versions".

File: input/pagecontent/conformance-profiles.md
Comment (saulakravitz) [inline input/pagecontent/conformance-profiles.md:6]: The file conformance-general.md hasn't been updated yet, right?

File: input/pagecontent/conformance-profiles.md
Comment (saulakravitz) [inline input/pagecontent/conformance-profiles.md:33]: SOme of the new elements are broken links.

File: input/pagecontent/group-disease.md
Comment (saulakravitz) [inline input/pagecontent/group-disease.md:42]: If the modeling of Staging and Risk Assessments are supposed to be analogous/parallel, I'm not convinced that the FHIR profiles are analogous/parallel.

File: input/pagecontent/index.md
Comment (saulakravitz) [inline input/pagecontent/index.md:1]: I think this page should include a reference to CodexRT that builds on the mCODE RT support, and involves some of the same folks credited with the RT support in mCODE.

File: input/pagecontent/profiles.md
Comment (saulakravitz) [inline input/pagecontent/profiles.md:7]: I recommend that the profiles, valuesets, etc pages are a waste of time to maintain and of very low utility to the reader.
Better to create groups in the sushi-config.yaml along the same lines (e.g., group the maturity level 2 and maturity level 0 profiles into separate groups in the sushi-config) and then have the menu link to sections of the artifacts.html page.   These pages, since they don't contain the descriptions of the resources, just their name, are MUCH LESS USEFUL TO THE READER than the content of the artifacts.html page.

File: input/pagecontent/basic_examples.md
Comment (saulakravitz) [inline input/pagecontent/basic_examples.md:1]: SOmething is going on in this file... All of the text turned blue.

**Comment by saulakravitz** [comment]
Regarding:  https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/ped-cancer/CodeSystem-snomed-requested-cs.html

USCRS-34947 | Surveillance Epidemiology and End Results Program Extent of Disease, 2018 edition (tumor staging)
-- | --   ==> Issues in US Edition only -- 761681000124102 |Surveillance Epidemiology and End Results Program, Extent of Disease - 2018 edition (tumor staging)|.   Note that mCODE STU3 depends on INt'l edition of SNOMED.
USCRS-34949 | Enneking surgical staging system (tumor staging) ==> Term request was rejected by US CRM.
USCRS-34955 | UICC Classification of Malignant Tumours 8th Edition (tumor staging)
USCRS-34956 | UICC Classification of Malignant Tumours 7th Edition (tumor staging)
    ==> Both of these are "This request is still on hold by SNOMED International pending a discussion with UICC."

**Comment by saulakravitz** [comment]
During STU3 ballot reconciliation we agreed to "start the process for getting an official codesystem for TG263".  This was a stupid committment.   I've sidestepped this in my other IGs where a similar problem was raised by creating a local codesystem, without any codes, just as a URI to hang the concept map off of.  Recommend you do the same here.

**Review by saulakravitz — APPROVED** [review]
See my comments sprinkled throughout.
They (obviously) need not be addressed before doing this merge, but figured I'd write them down so they'd be available in a convenient place.

**Pull request merged**
